web/packages/teleport/src/Integrations/status/AwsRa/AwsRaDashboard.tsx

```diff
@@ -0,0 +1,217 @@
+    </>
+  );
+}
+
+function AwsRaTitle({ integration }: { integration: IntegrationAwsRa }) {
+  const theme = useTheme();
+  const history = useHistory();
+  const integrationOps = useIntegrationOperation();
+  const { status, labelKind } = getStatusAndLabel(integration);
+
+  const anchorIdReg = new RegExp('[^\\/]+$');
+  const trustAnchorId = integration?.spec?.trustAnchorARN?.match(anchorIdReg);
+
+  async function removeIntegration() {
+    await integrationOps.remove();
+    integrationOps.clear();
+    history.push(cfg.routes.integrations);
+  }
+
+  async function editIntegration(req: EditableIntegrationFields) {
+    await integrationOps.edit(req);
+    integrationOps.clear();
+  }
+
+  return (
+    <Flex mt={3} justifyContent="space-between" alignItems="center" mb={4}>
+      <Flex alignItems="center" data-testid="aws-oidc-title">
+        <HoverTooltip placement="bottom" tipContent="Back to integrations">
+          <ButtonIcon
+            as={InternalLink}
+            to={cfg.routes.integrations}
+            aria-label="back"
+          >
+            <ArrowLeft size="medium" />
+          </ButtonIcon>
+        </HoverTooltip>
+        <Flex flexDirection="column" ml={1}>
+          <Flex alignItems="center" gap={2}>
+            <Text bold fontSize={6}>
+              {integration.name}
+            </Text>
+            <Label kind={labelKind} aria-label="status" px={3}>
+              {status}
+            </Label>
+          </Flex>
+          <Flex gap={1}>
+            Trust Anchor ARN:{' '}
+            {trustAnchorId && trustAnchorId.length === 1 ? (
+              <Link
+                target="_blank"
+                href={`https://console.aws.amazon.com/rolesanywhere/home/trust-anchors/${trustAnchorId[0]}`}
+              >
+                <Text
+                  style={{
+                    fontFamily: theme.fonts.mono,
+                  }}
+                >
+                  {integration.spec?.trustAnchorARN}
+                </Text>
+              </Link>
+            ) : (
+              <Text
+                style={{
+                  fontFamily: theme.fonts.mono,
+                }}
+              >
+                {integration.spec?.trustAnchorARN}
+              </Text>
+            )}
+          </Flex>
+        </Flex>
+      </Flex>
+      <Flex gap={1} alignItems="center">
+        <MenuButton icon={<Icons.Cog size="small" />}>
+          <MenuItem
+            onClick={() =>
+              history.push(
+                cfg.getIntegrationEnrollRoute(IntegrationKind.AwsRa, 'access'),
+                {
+                  integrationName: integration.name,
+                  trustAnchorArn: integration.spec.trustAnchorARN,
+                  syncProfileArn: integration.spec.profileSyncConfig.profileArn,
+                  syncRoleArn: integration.spec.profileSyncConfig.roleArn,
+                  edit: true,
+                }
+              )
+            }
+          >
+            Edit...
```

Comment:
```suggestion
            Edit…
```

super ultra turbo pedantic nit, but we try to use the ellipsis unicode character rather than three periods. same below in Delete